tools/site_and_regional/run_neon.py

```diff
@@ -747,7 +754,7 @@ def parse_neon_listing(listing_file, valid_neon_sites):
 
             tmp_df = tmp_df[tmp_df[7].str.contains(latest_version)]
             # -- remove .nc from the file names
-            tmp_df[9] = tmp_df[9].str.replace(".nc", "")
+            tmp_df[9] = tmp_df[9].str.replace(".nc", "", regex=False)
```

Comment:
For education I'd love to hear about the reason for this. Where there some cases that were failing that requires this addition?

Reply:
Not failing, Python was giving a warning:

`[FutureWarning: The default value of regex will change from True to False in a future version](https://stackoverflow.com/questions/66603854/futurewarning-the-default-value-of-regex-will-change-from-true-to-false-in-a-fu)`

I don't think we actually need this to be a regex evaluation?


according to: https://pandas.pydata.org/docs/whatsnew/v1.2.0.html#deprecationse, I updated this function accordingly. I don't think we need

Reply:
Awesome, that is a great future protection to have. Thanks for bringing this in. Resolving this thread...